nss-day-cohort-43/nuttyshell-react-plentyofwitches
# Bl delete messages

Testing Instructions:
-Clear out all of your messages from the database
-Login as a user, add a few messages, and try deleting some of them
-Login as a different user, make sure you can't delete the messages of the other user, and try adding and deleting messages as that user
-Try and break it!
